### PR TITLE
feat:added bookmark

### PR DESCRIPTION
**Pull request by deepak29-git**
1. add to bookmark
2. remove from bookmark

image

### CODE REVIEW

**Review by ragini07 — APPROVED** [review]
Clean code
Good naming convention

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import "./App.css";
-
 import { Navbar } from "./Components/Navbar/Navbar";
 import { Home } from "./Pages/Home/Home";
 import { SignIn } from "./Pages/Signin/Signin";
 import { Routes, Route } from "react-router-dom";
 import { Signup } from "./Pages/Signup/Signup";
 import { useAuth } from "./Context/auth-context";
 import { PageNotFound } from "./Pages/PageNotFound/PageNotFound";
+import { Bookmark } from "./Pages/Bookmark/Bookmark";
```
Comment (vikrantkode) [inline src/App.js:9]: try import from one js file...code looks cleaner

**Review by vikrantkode — APPROVED** [review]
one suggestion left .. have a look on them
 - Ui is clean...'
 - good to go

File: src/App.js (hunk 1 above)
Comment (deepak29-git) [reply src/App.js:9]: ok

**Pull request merged**